rwcancel/select_default.go

```diff
@@ -10,5 +10,6 @@ package rwcancel
 import "golang.org/x/sys/unix"
 
 func unixSelect(nfd int, r *unix.FdSet, w *unix.FdSet, e *unix.FdSet, timeout *unix.Timeval) error {
-	return unix.Select(nfd, r, w, e, timeout)
+	_, err := unix.Select(nfd, r, w, e, timeout)
```

Comment:
I guess x/sys/unix changed/unified the API across platforms? Or was this always broken?

Reply:
This was always broken, but because of the way build tags have been used so far this rwcancel package has never been built on macOS when building the device package. (I guess no one else is running `go test ./...`.)

With the conn/device split I do in another commit, the rwcancel package gets built on macOS as part of the device package build (but it all gets thrown away by the linker iirc).